docs/install/install-aur.md

```diff
@@ -6,6 +6,34 @@ sidebar_position: 4
 
 # Installing Vignette using AUR
 
-something
+If you wish to install Vignette on your Arch Linux computer, you may use any AUR helper of you choice. The package is [`vignette-git`](https://aur.archlinux.org/packages/vignette-git/). In the example bellow, i will be using yay.
 
--both-
+
+
+## Step 1: Acces your terminal
+
+You may use any terminal emulator of your choice.
+
+
+
+## Step 2: Install Vignette
+
+Please type `yay -S vignette-git` and press the `Enter` key. 
+
+![](/img/install/Konsole-yay.png)
+
+You should select the right package, in this case by typing `1`.
+
+![](/img/install/yay-select.png)
+
+You may proceed with the installation by typing `y`.
+
+![](/img/install/yay-yes.png)
```

Comment:
Explain only the selection of the right package. The person knows the terminal, they know how to say yes in a terminal.